magento/magento2
# 7399-clickableoverlay-option-doesnot-work

### Description
Fix has been tested  in Magento 2.1, Magento 2.2, Magento 2.3. Also tested in Mobile Device for same.

### Fixed Issues (if relevant)

1. magento/magento2#7399: Clickable Overlay option doesn't work has been fixed


### Manual testing scenarios

1. Make the changes in the file modal.js via PR
2. Perform php bin/magento s:s:d
3. Perform php bin/magento c:f
5. Run script to check

```
require(['jquery','Magento_Ui/js/modal/modal'],` function($, modal) {
   $('<div>Overlay content</div>').modal().modal('openModal');
});
```

6. Now when you click on overlay it will close the modal as default `clickableOverlay` value is `true`.

### Contribution checklist
